src/psyclone/domain/lfric/arg_ordering.py

```diff
@@ -243,12 +245,13 @@ def append_array_reference(self, array_name, indices, intrinsic_type,
         :param indices: the indices to be used in the PSyIR reference. It \
             must either be ":", or a PSyIR node.
         :type indices: List[Union[str, py:class:`psyclone.psyir.nodes.Node`]]
-        :param str intrinsic_type: the intrinsic type of the array, must \
-            be one of "real", "integer", or "logical"
+        :param intrinsic_type: the intrinsic type of the array.
+        :type intrinsic_type: \
+            :py:class:`psyclone.psyir.symbols.datatypes.ScalarType.Intrinsic`
         :param tag: optional tag for the symbol.
         :type tag: Optional[str]
         :param symbol: optional the symbol to use.
-        :rtype: :py:class:`psyclone.psyir.symbols.Symbol
+        :rtype: :py:class:`psyclone.psyir.symbols.Symbol`
```

Comment:
Blank line and a `:returns:` too please. Actually, this is also missing a `:type symbol: Optional[:py:class:`psyclone.....`]`

Reply:
Actually ... the `:returns;` and :`rtype;` was there, in the line below. The `:rtype:` should have been `:type symbol:` (plus an optional). All fixed, thanks.

Reply:
Oh yes, I completely failed to see that. Thanks for fixing.